idaes/ui/fsvis/static/js/stream_table.js

```diff
@@ -36,7 +36,17 @@ export class StreamTable {
             if (column_header !== "") {
                 // If the column_header is Variable then we don't want the column to be right-aligned and we want the column to be pinned to the left so when the user scrolls the column scrolls with them
                 if (column_header === "Variable") {
```

Comment:
This is where you could take the plain text sent from the server and make it into appropriate HTML, e.g. if the server sent "VariableName,m/s^2" you could split on the comma and format.